.internal-ci/helm/fog-services/templates/fog-view-rollout-cr.yaml

```diff
@@ -192,21 +193,22 @@ spec:
             app: fog-view-store
             {{- include "fogServices.labels" . | nindent 12 }}
         spec:
+          # TODO: constraints, affinity used fog-view; but selector used fog-view-store?
           topologySpreadConstraints:
           - topologyKey: topology.kubernetes.io/zone
             maxSkew: 1
             whenUnsatisfiable: DoNotSchedule
             labelSelector:
               matchLabels:
-                app: fog-view
+                app: fog-view-store
                 helm.sh/chart: {{ include "fogServices.chart" . }}
                 {{- include "fogServices.selectorLabels" . | nindent 16 }}
```

Comment:
I think this is correct, we want to make sure the store replicas are distributed across zones for availability.

Reply:
Those specifics are more for review by @aweis89 . I'm just cleaning up ports, names, etc.